include/stellar/io/import_sequence.hpp

```diff
@@ -0,0 +1,163 @@
+#pragma once
+
+#include <seqan/seq_io.h>
+
+namespace stellar
+{
+
+using namespace seqan2;
+
+template <typename TId>
+inline bool
+_checkUniqueId(std::set<TId> & uniqueIds, TId const & id)
+{
+    TId shortId;
+    typedef typename Iterator<TId const>::Type TIterator;
+
+    TIterator it = begin(id);
+    TIterator itEnd = end(id);
+
+    // (cut at first whitespace)
+    while (it != itEnd && *it > ' ')
+    {
+        appendValue(shortId, *it);
+        ++it;
+    }
```

Comment:
I personally would write this as a for loop (The typedef above is also not needed):
```suggestion

    // (cut at first whitespace)
    for (auto it = begin(id); it != itEnd && *it > ' '; ++it)
    {
        appendValue(shortId, *it);
    }
```